native/arguments.go

```diff
@@ -79,31 +79,34 @@ type UserFileArguments struct {
 	ArgumentsFile string
 }
 
-// Configure returns the inputArgs plus the additional arguments specified by the end user through the file, preference given to user arguments
+// Configure returns the inputArgs plus the additional arguments provided via argfile, setting via the '@argfile' format
 func (u UserFileArguments) Configure(inputArgs []string) ([]string, string, error) {
+
 	rawArgs, err := ioutil.ReadFile(u.ArgumentsFile)
 	if err != nil {
 		return []string{}, "", fmt.Errorf("read arguments from %s\n%w", u.ArgumentsFile, err)
 	}
 
-	parsedArgs, err := shellwords.Parse(string(rawArgs))
-	if err != nil {
-		return []string{}, "", fmt.Errorf("unable to parse arguments from %s\n%w", string(rawArgs), err)
+	fileArgs := strings.Split(string(rawArgs), "\n")
```

Comment:
>The arguments within a file can be separated by spaces or new line characters.

This isn't totally clear to me. Does it mean you can pick one? Either spaces or new lines? or does it mean you can mix them in the same file? Like one of the two between each argument? 

I feel like we are probably ok only supporting either one or the other. I do worry a little bit about what would happen if you mixed the two. It would try to parse using EOL characters and probably have some weird variation.

We might want to clarify in the docs that buildpacks expects one or the other, but that you need to stick with one consistently throughout the argfile.

Reply:
At the moment the PR supports one **_or_** the other. 

> The arguments within a file can be separated by spaces or new line characters.

I tested what this meant as it is unclear, and it appears to support a mix. I agree we should state that one or the other is supported by the buildpack and document this.

However, I believe if we were to be presented with a mixed file, the separation on newlines would happen and ultimately the file would be either left as-is or re-written without `-jar` and with the contents joined as one single line, both of which should be fine. There is an edge case here for a mixed file where we could get:
```
-testarg
-testarg2 testarg3
-testarg4 -jar my-jar.jar
-testarg5
```
In this case we won't match the `-jar` as a needle in the haystack. So if I'm correct in the above, we'd be supporting one or the other but also mixed should happen to work too, in most cases?

Reply:
I think that's fair. You could also say we *recommend* sticking with spaces or EOLs, but that you can mix at your own risk.